### ISSUE TITLE
Taskrun replay is not working

### ISSUE THREAD

**Issue by brian-mulier-p** [kestra-io/kestra#1209]
### Expected Behavior

When we click on replay on a taskrun the given taskrun should replay

### Actual Behaviour

It greys out after clicking without doing anything (even after refresh). Also an error appears in console
![image](https://user-images.githubusercontent.com/37618489/234879823-660a03f1-68f6-4c15-856f-77fe10929f20.png)

### Steps To Reproduce

On an execution with a failed taskrun, try to click on replay button

### Environment Information

- Kestra Version: 0.9.0-SNAPSHOT

### Example flow
```yaml
id: "fail"
namespace: "some.test"
tasks:
  - id: failing_task
    type: io.kestra.core.tasks.executions.Fail
```

**Comment by loicmathieu** [comment]
I cannot reproduce it with the given flow.

@brian-mulier-p can you try it again with latest master and a fresh UI build ?

**Comment by brian-mulier-p** [comment]
The issue was happening only when you have your UI in french + not running it under npm but as the raw app since source map is making it work but displaying ugly {-id} message while raw app makes it fail with grey screen.
Fixed in commit 92231cef0f0e3f6306846847ffc619c22cd79e17
